packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/View/View.results.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { connect, ConnectedProps } from 'react-redux'
+import { LinkContainer } from 'react-router-bootstrap'
+import { bindActionCreators } from 'redux'
+import styled from 'styled-components'
+
+import { NftAsset } from '@core/network/api/nfts/types'
+import { Image, Text } from 'blockchain-info-components'
+import { actions, selectors } from 'data'
+import { media } from 'services/styles'
+
+const NftViewResults: React.FC<Props> = (props) => {
+  const { asset, nftsActions } = props
+  const AssetImg = styled.img`
```

Comment:
You'll want to move these styled components outside of the NftViewResults component so they're not expensively recreated every time this component is loaded.